package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@angular-eslint/angular-eslint",
   "private": true,
   "volta": {
-    "node": "16.19.0",
+    "node": "16.20.0",
```

Comment:
@JamesHenry any rules/guidelines around updating the Volta config?

Reply:
For minor bumps feel free to merge them, for major ones let's discuss in advance please!